pages/download.html

```diff
@@ -206,7 +207,7 @@ <h2><img src="{% static '/static/images/download_mac.png' %}" class="feature_ico
     <p>
       <a href="{{ stable_osxintel }}"
          onClick="javascript:trackDownload('{{ stable_osxintel_analytics_conversion }}');">
-        {% blocktrans %}Download {{ stable_long_version }} for Mac OS X 10.5+ (Intel){% endblocktrans %}</a>
+        {% blocktrans %}Download {{ stable_long_version }} for macOS 10.5+ (Intel){% endblocktrans %}</a>
```

Comment:
Is this still correct? What is the oldest version of macOS still supported?

Reply:
> Is this still correct? What is the oldest version of macOS still supported?

10.8